packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2956,6 +2967,18 @@ class InputDecoration {
   /// Only one of [error] and [errorText] can be specified.
   final String? errorText;
 
+
+  /// Callback for building the error widget.
+  ///
+  /// If non-null, [errorText] will be passed to this builder,
+  /// and the returned widget will be displayed in place of [error].
+  ///
+  /// Use [errorBuilder] instead of [error] if you need to show the
+  /// validator error but at the same time also customize the error widget.
+  ///
+  /// Only one of [error] or [errorBuilder] can be specified.
```

Comment:
Somewhere I think we should mention that errorText comes from TextFormField.validator. Is here the best place?

See also:

  * [TextFormField.validator], which passes its validation error as [InputDecoration.errorText] through to `errorBuilder`.

Reply:
This is good now 👍